src/hdnode.js

```diff
@@ -51,7 +51,7 @@ HDNode.fromSeedHex = function (hex, network) {
   return HDNode.fromSeedBuffer(new Buffer(hex, 'hex'), network)
 }
 
-HDNode.fromBase58 = function (string, networks) {
+HDNode.fromBase58 = function (string, networks, skipValidation) {
```

Comment:
I don't like adding arguments like this because it can cause weird results if people have bound these functions in a way that might currently ignore the extra arguments (`.forEach(fromBase58)` or something),

But,  I can't currently see another way around this...